.github/workflows/update-ic.yml

```diff
@@ -16,22 +16,35 @@ jobs:
       - name: Check new ic version
         id: update
         run: |
-          current_ic_commit=$(cat ./src/canister_tests/Cargo.toml | sed -n 's/.*{\s*git\s*=\s*"https:\/\/github\.com\/dfinity\/ic",\s*rev\s*=\s*"\([a-z0-9]*\)" }.*/\1/p' | head -n 1)
-          echo "current ic commit '$current_ic_commit'"
-          latest_ic_commit=$(curl --silent -H 'Accept: application/vnd.github.v3+json' https://api.github.com/repos/dfinity/ic/commits/master | jq -cMr .sha)
-          echo "latest ic commit '$latest_ic_commit'"
-
-          if [ "$current_ic_commit" != "$latest_ic_commit" ]
-          then
+          shopt -s globstar
```

Comment:
After squinting at those changes for a bit, I'm guessing this is now performing the update in every cargo toml, since we now have more than one that depends on IC deps. Is that right?